packages/tasks/src/local-apps.ts

```diff
@@ -103,6 +103,23 @@ export const LOCAL_APPS = {
 		displayOnModelPage: isGgufModel,
 		deeplink: (model) => new URL(`https://backyard.ai/hf/model/${model.id}`),
 	},
+	jellybox : {
+		prettyLabel: "Jellybox",
+		docsUrl: "https://jellybox.com",
+		mainTask: "text-generation",
+		displayOnModelPage: (model) =>
+			isGgufModel(model) ||
+			model.library_name === "diffusers" && model.tags.includes("safetensors") && (model.pipeline_tag === "text-to-image" || model.tags.includes("lora")),
+		deeplink: (model) => {
+			if(isGgufModel(model)) {
+				return new URL(`jellybox://llm/models/huggingface/LLM/${model.id}`);
```

Comment:
This will create for example `jellybox://llm/models/huggingface/LLM/microsoft/Phi-3-mini-4k-instruct-gguf`  
for me the deeplink opens Jellybox but doesn't select the model to download 
<img width="1041" alt="Screenshot 2024-05-30 at 16 27 41" src="https://github.com/huggingface/huggingface.js/assets/468620/0752a53f-7e32-4c09-9773-08f8281061c8">

It seems to be working in your video but with an internal id. Can you update the format?

Reply:
It's fixed in the latest update!
I was using the id from this previously.https://github.com/huggingface/huggingface.js/blob/f94376b3b763d53e4f54055c06d6ed34d219b003/packages/hub/src/lib/list-models.ts#L43

I'm currently working on converting the icon to a vector. Will reply here with the SVG once it's ready.

https://github.com/huggingface/huggingface.js/assets/22720475/04ea7f84-fb7d-4fc2-87e1-b34763c057e3

Reply:
It works well with that version. Thanks!